zookeeper-server/src/test/java/org/apache/zookeeper/ClientCnxnSocketFragilityTest.java

```diff
@@ -0,0 +1,310 @@
+package org.apache.zookeeper;
```

Comment:
LICENSE ?

Reply:
Thanks for comment. Accept it. I will add LICENSE statement in the test case.